homeassistant/components/daikin/sensor.py

```diff
@@ -40,7 +69,7 @@ def __init__(self, api, monitored_state) -> None:
     @property
     def unique_id(self):
         """Return a unique ID."""
-        return f"{self._api.mac}-{self._device_attribute}"
+        return f"{self._api.device.mac}-{self._device_attribute}"
 
```

Comment:
```suggestion
    @property
    def device_class(self) -> Optional[str]:
        """Return the class of this device."""
        return self._sensor[CONF_CLASS]

```

Reply:
Thanks for the suggestion, I've added this property directly in the class Sensors (nothing in `const.py`)